src/MockFestivalsApiClient.php

```diff
@@ -17,11 +18,11 @@
 class MockFestivalsApiClient extends FestivalsApiClient
 {
 
-    protected array $called_with = [];
+    private array $called_with = [];
```

Comment:
Although this is just a mock class, technically this is a BC break since we don't officially tell people not to extend this.

I think possibly best to leave these properties `protected`, and add a docblock 

```
/**
 * @final since 2.4.1
 */
```

To the class itself to tell people not to extend it, we can potentially then make it an actual `final` class in v3.0.